### PR TITLE
[MPS] Add softplus backward

### PR DESCRIPTION
**Pull request by qqaatw**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: aten/src/ATen/native/mps/operations/Activation.mm [hunk 1]
```diff
@@ -1570,8 +1570,103 @@ Tensor glu_backward_mps (const Tensor& grad_output,
...
+              MPSGraph* mpsGraph = make_mps_graph();
+              newCachedGraph = new CachedGraph(mpsGraph);
+              MPSGraphTensor *gradOutputTensor = mpsGraphRankedPlaceHolder(mpsGraph, grad_output);
+
+              MPSGraphTensor *inputTensor = mpsGraphRankedPlaceHolder(mpsGraph, self);
 
+              MPSGraphTensor* unitTensor = [mpsGraph constantWithScalar:1.0
+                                                                  shape:@[@1]
+                                                               dataType:getMPSDataType(self.scalar_type())];
+              MPSGraphTensor* betaTensor = [mpsGraph constantWithScalar:beta.to<double>()
```
Comment (kulinseth) [inline aten/src/ATen/native/mps/operations/Activation.mm:1620]: If you are passing beta as a `constantWithScalar` this needs to be in the key.

Comment (kulinseth) [inline aten/src/ATen/native/mps/operations/Activation.mm:1620]: Also there is no need to have this as a constant, you can pass it as a Placeholder.

**Comment by kulinseth** [comment]
@qqaatw , can you please create a followup PR with above requests.

**Comment by qqaatw** [comment]
@kulinseth Ok, no prob.

File: aten/src/ATen/native/mps/operations/Activation.mm (hunk 1 above)
Comment (qqaatw) [reply aten/src/ATen/native/mps/operations/Activation.mm:1620]: I see the `beta` in `softplus_out_mps` is also regarded as a constant, do you want me to make it a Placeholder as well?

Comment (kulinseth) [reply aten/src/ATen/native/mps/operations/Activation.mm:1620]: Yes, please.

**Pull request closed without merging**